performancetest/src/main/java/org/corfudb/performancetest/StreamPerformanceTest.java

```diff
@@ -0,0 +1,103 @@
+package org.corfudb.performancetest;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.view.stream.IStreamView;
+import org.junit.Test;
+import java.io.IOException;
+
+/**
+ * Created by Nan Zhang and Lin Dong.
+ */
+
+public class StreamPerformanceTest extends PerformanceTest {
+    static MetricRegistry metricRegistry;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)